wadas/ui/configure_ftp_cameras_dialog.py

```diff
@@ -268,6 +276,27 @@ def validate(self):
         else:
             self.ui.label_errorMessage.setText("No server port provided!")
             valid = False
+        start_passive_port = self.ui.lineEdit_passive_port_range_start.text()
+        end_passive_port = self.ui.lineEdit_passive_port_range_end.text()
+        if not start_passive_port:
+            self.ui.label_errorMessage.setText("Start passive port not provided!")
+            valid = False
+        elif start_passive_port and (int(start_passive_port) < 1 or int(start_passive_port) > 65535):
```

Comment:
elif ((start_passive_port_number := int(start_passive_port)) < 1 or start_passive_port_number > 65535:

Reply:
if we remove **elif start_passive_port and ...** it will fail on empty string

Reply:
The elif is executed only if start_passive_port is a string which is not empty.